scripts/cluster_network.py

```diff
@@ -377,6 +426,8 @@ def consense(x):
                                                line_length_factor=line_length_factor,
                                                potential_mode=potential_mode,
                                                solver_name=snakemake.config['solving']['solver']['name'],
+                                               algorithm=snakemake.config.get('clustering', {}).get('cluster_network', {}).get('algorithm', 'kmeans'),
+                                               feature=snakemake.config.get('clustering', {}).get('cluster_network', {}).get('feature', None),
```

Comment:
```suggestion
                                               algorithm=cluster_config.get('algorithm', 'kmeans'),
                                               feature=cluster_config.get('feature', None),
```

You could move the common expression:

```py
snakemake.config.get('clustering', {}).get('cluster_network', {})
```

further up between `custom_busmap = ` and `clustering = `:

```py
cluster_config = snakemake.config.get('clustering', {}).get('cluster_network', {})
```